File: tests/fixtures/implements_collision/scope/csharp/two.cs
using Alias = A.Real.Target;

namespace A
{
    // Nothing here is called Alias, so the file-scope alias is what it means.
    public class AChild : Alias { }
}

namespace B
{
    // C# looks in the enclosing namespace before its using aliases, so this
    // Alias is the one, not the file-scope rename.
    public interface Alias { }

    public class BChild : Alias { }
}

File: tests/fixtures/implements_collision/scope/rust/two.rs
pub mod a {
    use crate::api::Target as Alias;
    use external::Target as Outside;

    pub struct AChild;

    impl Alias for AChild {}

    pub struct AOutside;

    impl Outside for AOutside {}
}

pub mod b {
    // A rename inside a sibling module is not in scope here.
    pub trait Alias {}

    pub struct BChild;

    impl Alias for BChild {}
}
